sdk/python/feast/feature_store.py

```diff
@@ -275,7 +280,8 @@ def _list_feature_views(
             ):
                 fv.entities = []
                 fv.entity_columns = []
-            feature_views.append(fv)
+            if all(fv.tags.get(key, None) == val for key, val in tagDict.items()):
```

Comment:
@tokoko later we can move this logic in a global function like:
```py
def has_all_tags(resource_tags: Dict[str, str], requested_tags: Dict[str, str]) -> bool:
...
```
in the `utils` module? I see at least 2 copies of the same logic in this PR and others will follow when we check for authorized permissions (against all the managed objects, not just `FeatureView`s)

Reply:
We won't need a separate `list` permission that way, right? If a user has a `read` permission on every resulting object, that will imply a list permission as well...

Reply:
this has been moved to a global function in `utils`